portable/GCC/ARM_CM23_NTZ/non_secure/portasm.c

```diff
@@ -154,8 +153,7 @@
             "   ldr  r1, [r2]                                   \n" /* Read pxCurrentTCB. */
             "   ldr  r0, [r1]                                   \n" /* Read top of stack from TCB - The first item in pxCurrentTCB is the task top of stack. */
             "                                                   \n"
-            "   ldm  r0!, {r1-r2}                               \n" /* Read from stack - r1 = PSPLIM and r2 = EXC_RETURN. */
-            "   msr  psplim, r1                                 \n" /* Set this task's PSPLIM value. */
+            "   ldr  r2, [r0]                                   \n" /* Read from stack - r2 = EXC_RETURN. */
```

Comment:
After excuting the original code: ldm r0!, {r1-r2}, r0 points t0 the address EXC_RETURN + 4, while "ldr r2, [r0]" points to exactly EXC_RETURN. So, use "ldm r0!, {r2}" or "adds r0, #4" would be better.

Reply:
Does this comment apply to the latest changes?